src/main/java/core/basesyntax/strategy/impl/Balance.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.exception.InvalidValueExeption;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationAnalysis;
+
+public class Balance implements OperationAnalysis {

```

Comment:
```suggestion
public class BalanceOperation implements OperationAnalysis {
```
please rename all other operations like in example above